venues/NeurIPS.cc/2021/Conference/python/build-conflicts.py

```diff
@@ -0,0 +1,229 @@
+import openreview
+import argparse
+
+
+def get_profile_info(profile):
+    
+    domains = set()
+    emails=set()
+    relations = set()
+    publications = set()
+    common_domains = ['gmail.com', 'qq.com', '126.com', '163.com', 
+                      'outloook.com', 'hotmail.com', 'yahoo.com', 'aol.com', 'msn.com', 'ymail.com', 'googlemail.com', 'live.com']
+
+    ## Institution section, get history within the last three years
+    for h in profile.content.get('history', []):
+        if h.get('end') is None or int(h.get('end')) > 2017:
+            domain = h.get('institution', {}).get('domain', '')
+            domains.update(openreview.tools.subdomains(domain))
+
+    ## Relations section, get coauthor/coworker relations within the last three years + all the other relations
+    for r in profile.content.get('relations', []):
+        if r.get('relation', '') in ['Coauthor','Coworker']:
+            if r.get('end') is None or int(r.get('end')) > 2017:
+                relations.add(r['email'])
+        else:
+            relations.add(r['email'])
+    
+    ## Emails section        
+    for email in profile.content['emails']:
+        emails.add(email)
+        # if institution section is empty, add email domains
+        if not domains: 
+            domains.update(openreview.tools.subdomains(email))
+            
+    ## Publications section: get publications within last three years
+    for pub in openreview.tools.iterget_notes(client, content={'authorids': profile.id}):
+        if pub.content.get('_bibtex','') and int(pub.content['_bibtex'].split('year={')[1][:4]) > 2017:
```

Comment:
note.tcdate should have the same value than _bibtex, I would use tcdate  because not all the publications have a _bibtex entry.

Reply:
openreview.tools.iterget_notes(client, content={'authorids': '~Thomas_Leung1'})
this note.cdate is None

Can we use note.tcdate instead?

Reply:
Yes, you can check cdate first and then tcdate if cdate is not present. 

Generally all the submissions notes have cdate == tcdate with the exception of DBLP publications where cdate is the publication date and tcdate is the date when the publication was uploaded to OpenReview.

Reply:
okay, thanks. I've made the changes.